### PR TITLE
Multiplatform setup POC

### PR DESCRIPTION
**Pull request by Egorand**

### CODE REVIEW

File: common/src/main/kotlin/okio/System.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okio
+
+expect fun arraycopy(src: ByteArray, srcPos: Int, dest: ByteArray, destPos: Int, length: Int)
```
Comment (swankjesse) [inline common/src/main/kotlin/okio/System.kt]: internal ?

Comment (Egorand) [reply common/src/main/kotlin/okio/System.kt]: Yeah, I wonder if it's gonna work as expected, since technically those are different modules

Comment (JakeWharton) [reply common/src/main/kotlin/okio/System.kt]: Not a bug. You can't cross modules with `internal`.

Comment (Egorand) [reply common/src/main/kotlin/okio/System.kt]: Makes sense, hence we should leave it public and ensure it's not exposed as a transitive dependency

Comment (JakeWharton) [reply common/src/main/kotlin/okio/System.kt]: Why are we adding a transitive dependency instead of making the okio module multiplatform?

Comment (Egorand) [reply common/src/main/kotlin/okio/System.kt]: Good point, I guess a better setup would be `okio-common` and `okio` itself being the JVM module

Comment (JakeWharton) [reply common/src/main/kotlin/okio/System.kt]: I've been using nested projects for multiplatform if that interests you:
```
okio/
  js/
  jvm/
  src/
```

**Pull request merged**